Python/correlation/test/test_vary_frames.py

```diff
@@ -0,0 +1,83 @@
+"""
+Provides the possibility to change frames.
+Plot the signals together with the correlated signals, and find the accuracy of the result.
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+
+import Python.correlation.correlation as correl
+import Python.correlation.tdoa as tdoa
+
+import Python.signal_generation.positioning as sg_pos
+import Python.signal_generation.receiver as sg_rec
+import Python.signal_generation.source as sg_src
+
+
+#import correlation.correlation
+
+#@pytest.mark.plot
+def test_plot_of_signals_and_correlation_signal():
+    # Set variables:
+    sample_frequency = 100.0  # [kHz]
+    pulse_length = 1.0  # [ms]
+    signal_length = 400  # [samples]
+    frame = [0, 400]  # [start of signal, end of signal]
+
+    positions = np.array(
+        [
+            sg_pos.Position(x=0.0, y=0.0, z=0.0),
+            sg_pos.Position(x=1.83, y=0.0, z=0.0),
+            sg_pos.Position(x=0.0, y=0.91, z=0.91),
+            #sg_pos.Position(x=1.83, y=0.91, z=0.0),
+            #sg_pos.Position(x=1.0, y=0.0, z=0.91),
+        ]
+    )
+
+    hydro_array = sg_rec.HydrophoneArray(
+        positions=positions,
+    )
+
+    pinger_position_one = sg_pos.Position(
+        x=4.0,
+        y=0.0,
+        z=0.0,
+    )
+
+    pinger = [
+        sg_src.Pinger(
+            frequency=25.0,  # [kHz]
+            sampling_frequency=sample_frequency,  # [kHz]
+            period=30.0,
+            pulse_length=pulse_length,
+            use_window=True,
+            position=pinger_position_one,
+        ),
+    ]
```

Comment:
```suggestion
    pinger = sg_src.Pinger(
        frequency=25.0,  # [kHz]
        sampling_frequency=sample_frequency,  # [kHz]
        period=30.0,
        pulse_length=pulse_length,
        use_window=True,
        position=pinger_position,
    ),
```
There's not really a need to use a list of pingers here.